src/main/java/sawkiewicz/task1/ItemController.java

```diff
@@ -0,0 +1,63 @@
+package sawkiewicz.task1;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+
+@RestController
+public class ItemController {
+    ItemRepository itemRepository = new ItemRepository();
+    ShoppingCartRepository shoppingCartRepository = new ShoppingCartRepository();
+
+
+
+    @RequestMapping(method = RequestMethod.GET, value = "/total/{cartId}")
+    ResponseEntity<String> showTotalPrice(@PathVariable String cartId) {
+        return ResponseEntity.ok("Total price is: " + shoppingCartRepository.findById(cartId).calculateTotalCost());
```

Comment:
according to Rest specification, you should return an object like the one below

```
{
     "response": "Ok"
     "message": ""
}
```

Reply:
Please read about JSON https://www.copterlabs.com/json-what-it-is-how-it-works-how-to-use-it/